src/key-binder.c

```diff
@@ -28,15 +29,6 @@ struct _BriskKeyBinderClass {
 };
 
 /**
- * BriskKeyBinder is used to bind global x11 shortcuts
```

Comment:
Not really sure I wanted this as public API as I wanted to keep the BriskKeyBinder as a reusable opaque object.